coalib/parsing/filters/CanFixFilter.py

```diff
@@ -0,0 +1,14 @@
+from coalib.parsing.filters.Filter import filter
+
+
+@filter
+def can_fix_filter(bear, args):
+    """
+    Filters the bears by ``LANGUAGES``.
+
+    :param bear:        Bear object
+    :param args:        Set of fixes on which bear is to be filtered
+    :return:            True if this bear is to be displayed for given
+                        args, False otherwise
```

Comment:
missing periods at end of sentences :)
\+
Enclose True and False into double backticks, as these are code terms ;)
(applies for all filter docstrings)